src/ast_transform.coffee

```diff
@@ -844,8 +849,8 @@ do ()=>
           inner.need_nest = false
           # TODO clarify modifier's order
           for mod, idx in root.modifier_list
-            inner = fn_apply_modifier inner, mod, ctx , idx == 0 and root.modifier_list.length != 1
-          
+            not_need_nest = idx != root.modifier_list.length - 1
```

Comment:
If that means "do not nest last modifier" than I would suggest clarifying it like that:
`need_nest = idx == root.modifier_list.length - 1` and pass it like that into `fn_apply_modifier`